lighthouse-core/config/config.js

```diff
@@ -714,28 +690,38 @@ class Config {
    * @return {LH.Config['audits']}
    */
   static requireAudits(audits, configPath) {
-    if (!audits) {
+    const expandedAudits = Config.expandAuditShorthand(audits);
+    if (!expandedAudits) {
       return null;
     }
 
     const coreList = Runner.getAuditList();
-    return audits.map(auditDefn => {
-      if (!auditDefn.implementation) {
-        const path = auditDefn.path;
+    const auditDefns = expandedAudits.map(audit => {
+      let implementation;
```

Comment:
does this become `any`? or does `TS` infer based on the later assignments?

Reply:
it's `any` here, but by the time it's out of the if/else blocks it's `typeof Audit` from the assignments